<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWnd</class>
 <widget class="QWidget" name="MainWnd">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>822</width>
    <height>351</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QFrame" name="frame_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>501</width>
       <height>32</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QPushButton" name="ConfigDeviceButton">
      <property name="geometry">
       <rect>
        <x>4</x>
        <y>4</y>
        <width>61</width>
        <height>26</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Device parameters</string>
      </property>
      <property name="text">
       <string>CCD</string>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="PhotoButton">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>4</y>
        <width>51</width>
        <height>26</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Photo mode</string>
      </property>
      <property name="text">
       <string>Photo</string>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="VideoButton">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>4</y>
        <width>51</width>
        <height>26</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Video mode</string>
      </property>
      <property name="text">
       <string>Video</string>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="SessionsButton">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>4</y>
        <width>61</width>
        <height>26</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Session parameters</string>
      </property>
      <property name="text">
       <string>Sessions</string>
      </property>
     </widget>
     <widget class="QPushButton" name="HistogramButton">
      <property name="geometry">
       <rect>
        <x>268</x>
        <y>4</y>
        <width>41</width>
        <height>26</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Show histogram</string>
      </property>
      <property name="text">
       <string>Hist.</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="AutoApplyHistogram">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>7</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Auto apply histogram</string>
      </property>
      <property name="text">
       <string>hist.</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="ConfigButton">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>4</y>
        <width>61</width>
        <height>26</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Configuration</string>
      </property>
      <property name="text">
       <string>Config</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_FitToWindowScale">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>3</y>
        <width>31</width>
        <height>27</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Fit to window</string>
      </property>
      <property name="text">
       <string>[]</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_About">
      <property name="geometry">
       <rect>
        <x>467</x>
        <y>3</y>
        <width>31</width>
        <height>27</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>About...</string>
      </property>
      <property name="text">
       <string>?</string>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
     </widget>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QPushButton" name="SaveShotButton">
     <property name="toolTip">
      <string>Save to file</string>
     </property>
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="l_frames">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
       <horstretch>48</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>128</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>128</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="text">
      <string>sess frames</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QLabel" name="l_info">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
       <horstretch>100</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="text">
      <string>info</string>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QLabel" name="l_driver">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>100</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="text">
      <string>driver</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="5">
    <widget class="QFrame" name="ScrollFrame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>ConfigDeviceButton</tabstop>
  <tabstop>PhotoButton</tabstop>
  <tabstop>VideoButton</tabstop>
  <tabstop>SessionsButton</tabstop>
  <tabstop>HistogramButton</tabstop>
  <tabstop>AutoApplyHistogram</tabstop>
  <tabstop>ConfigButton</tabstop>
  <tabstop>pushButton_FitToWindowScale</tabstop>
  <tabstop>pushButton_About</tabstop>
  <tabstop>SaveShotButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
